setup.py

```diff
@@ -100,6 +100,7 @@
     "tifffile": ["tifffile"],
     "pyav": ["av"],
     "pillow-heif": ["pillow-heif"],
+    "rawpy": ["rawpy"]
```

Comment:
Do you know if there are additional installation steps beyond installing the python package?

Reply:
I did not have to do any additional installation steps. So, I think only installing the python package should be enough.